internal/service/davinci/resource_flow.go

```diff
@@ -2,968 +2,1367 @@ package davinci
+	)
+
+	flowVariablesDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"Returned list of Flow Context variables. These are variable resources that are created and managed by the Flow resource via `flow_json`.",
+	)
+
+	flowVariablesContextDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"The variable context.  Should always return `flow`.",
+	)
+
+	flowVariablesTypeDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"The variable's data type.  Expected to be one of `string`, `number`, `boolean`, `object`.",
+	)
+
+	flowVariablesMutableDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"A boolean that specifies whether the variable is mutable.  If `true`, the variable can be modified by the flow. If `false`, the variable is read-only and cannot be modified by the flow.",
+	)
+
+	flowVariablesMinDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"The minimum value of the variable, if the `type` parameter is set as `number`.",
+	)
+
+	flowVariablesMaxDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"The maximum value of the variable, if the `type` parameter is set as `number`.",
+	)
+
+	resp.Schema = schema.Schema{
+		// This description is used by the documentation generator and the language server.
+		Description: "Resource to import and manage a DaVinci flow in an environment.  Connection and Subflow references in the JSON export can be overridden with ones managed by Terraform, see the examples and schema below for details.",
+
+		Attributes: map[string]schema.Attribute{
+			"id": framework.Attr_ID(),
+
+			"environment_id": framework.Attr_LinkID(
+				framework.SchemaAttributeDescriptionFromMarkdown("The ID of the PingOne environment to import the DaVinci flow to."),
+			),
+
+			"name": schema.StringAttribute{
+				Description:         nameDescription.Description,
+				MarkdownDescription: nameDescription.MarkdownDescription,
+				Optional:            true,
+				Computed:            true,
+
+				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(attrMinLength),
+				},
 			},
-			"name": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Computed Flow Name after import. Matches 'name' in flow_json",
+
+			"description": schema.StringAttribute{
+				Description:         descriptionDescription.Description,
+				MarkdownDescription: descriptionDescription.MarkdownDescription,
+				Optional:            true,
+				Computed:            true,
 			},
-			"environment_id": {
-				Type:        schema.TypeString,
+
+			"flow_json": schema.StringAttribute{
+				Description: framework.SchemaAttributeDescriptionFromMarkdown("The DaVinci Flow to import, in raw JSON format.  Should be a JSON file that has been exported from a source DaVinci environment.  Must be a valid JSON string.").Description,
```

Comment:
The DaVinci Flow to import, in raw JSON format. Should be a JSON file of a single flow (without subflows) that has been exported from a source DaVinci environment.